docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -38,10 +38,11 @@ This table links to the release notes for each major release. These release note
 ==============================      =================================================
 Ansible Release                     Status
 ==============================      =================================================
-devel                               In development (2.10 unreleased, trunk)
+devel                               In development (2.11 unreleased, trunk)
+`2.10 Release Notes`_               Maintained (security **and** general bug fixes)
 `2.9 Release Notes`_                Maintained (security **and** general bug fixes)
 `2.8 Release Notes`_                Maintained (security **and** critical bug fixes)
-`2.7 Release Notes`_                Maintained (security fixes)
+`2.7 Release Notes`_                Unmaintained (end of life)
```

Comment:
I'm actually fuzzy on whether 2.7 goes unmaintained when ansible-base releases, or when Ansible releases?

Reply:
@samccann it's already gone unmaintained. It's been removed from CI and we've stopped cutting updates for it.